tools/labs/templates/hello_assignment/sol/Kbuild

```diff
@@ -0,0 +1,3 @@
+EXTRA_CFLAGS = -Wall -g -m32
```

Comment:
Do we need the extra sol directory?

Reply:
it should be called `hello-world`.
I assumed this as the source directory, it can have any name. `checker` directory is supposed to exist in order to be recursively copied to vm disdk